opm/autodiff/BlackoilWellModel.hpp

```diff
@@ -61,9 +61,14 @@
 
 namespace Opm {
 
+        template<typename TypeTag>
+        class BlackoilModelEbos;
+
         /// Class for handling the blackoil well model.
         template<typename TypeTag>
         class BlackoilWellModel {
+            // Needs acces to wells_ecl_
```

Comment:
`BlackoilWellModel` has access to `wells_ecl_`. 
```
 std::vector< const Well* > wells_ecl_;
```

I am sure whether there are some misunderstandings here.